Fix position of icon and highlighted text

JiwoonKim opened the pull request:
**Closes Issue** #36 
- vertically center the icon with the tooltip text
- replace blanklines Typography component with div element
- add more padding to highlighted text and fix font-size and position if necessary

**Applied Changes**
- replaced blanklines Typography component with div element
- refactored `<Tooltip />` into its own separate component
- vertically centered the icon for the tooltip text
- fixed height or rows and lines + added padding for highlighted text (for `types` in left panel)

**Result Screenshot**

<img src="https://user-images.githubusercontent.com/29671309/71441552-427df880-2745-11ea-8020-519ff2d1a366.png" width="500px" />

helfi92 commented on src/components/SchemaTable/index.jsx:
```diff
@@ -36,12 +36,19 @@ const useStyles = makeStyles(theme => ({
   descriptionColumn: {},
   line: {
+    display: 'flex',
+    alignItems: 'center',
```
It's centered on Chrome but on Firefox, it looks a bit off:

<img width="190" alt="Screen Shot 2019-12-27 at 8 59 14 AM" src="https://user-images.githubusercontent.com/3766511/71519864-45810200-2887-11ea-87e3-e89e62d759f5.png">

JiwoonKim replied:
Applied changes! Hope this is better 😃 

| Chrome |  Firefox |
|:---|:---|
| <img src="https://user-images.githubusercontent.com/29671309/71574777-9ca80080-2b2d-11ea-9207-b7113f639f1e.png" width="170px" /> | <img src="https://user-images.githubusercontent.com/29671309/71574734-5fdc0980-2b2d-11ea-8bae-647092e66679.png" width="170px" /> |

JiwoonKim commented:
> Weird, the issue is still there on Firefox. This is how it looks like:
> Are you able to replicate?

Hmmm... that's strange.
The icon seems to be placed in align with the text in my Firefox browser.

<img src="https://user-images.githubusercontent.com/29671309/71611563-42667880-2bdd-11ea-8ee0-2d8be6984589.png" width="100%" />

JiwoonKim commented:
Added some structure and style changes to the `Tooltip` component!
(also added a red line border to make it easier to compare icon position with the text, 
will remove it when this PR is approved 😄 )

This is what the changes applied look like in Firefox browser:
<img src="https://user-images.githubusercontent.com/29671309/71617676-7bafe000-2bff-11ea-84a9-75d22a3c601a.png" width="100%" />

Does it align correctly now?

> Also, there seems to be a console warning about a unique key. Just wanted to point this out. Feel free to fix this in a different PR.

I've also made some changes to fix this issue as well. There should be no console warning anymore :)

helfi92 commented on src/components/SchemaTable/index.jsx:
```diff
@@ -51,16 +51,24 @@ const useStyles = makeStyles(theme => ({
+  },
+  tooltip: {
+    fontSize: `${theme.typography.subtitle2.fontSize}`,
+    lineHeight: 1,
```
The `lineHeight` seems to have been the fix. Awesome work tracking this down! It was a tricky one.

helfi92 commented on src/components/Tooltip/index.jsx:
```diff
@@ -0,0 +1,54 @@
+          variant="subtitle2"
+          className={classes.tooltip}>
+          {`${keyword}: `}
+        </Typography>
```
When we're showing the icon, what do you think about removing the column and adding some margin (e.g., 4px instead)?

_Example: removing the column and adding some margin on propertyNames_

<img width="192" alt="Screen Shot 2019-12-31 at 4 42 36 PM" src="https://user-images.githubusercontent.com/3766511/71634757-cd9b3a80-2bec-11ea-94e1-245addd99d81.png">

JiwoonKim replied:
Great idea! Removed the colon from Tooltip's text.
It makes the UI look more cleaner 👍 

![image](https://user-images.githubusercontent.com/29671309/71637908-0e9a5a00-2c94-11ea-83d7-50265b9ab342.png)
